docs/source/changelog.rst

```diff
@@ -83,9 +93,7 @@ Improvements
   Added an improved differential equation generation method `'flat'` which improves performance by ~30%.
   This new method is currently not compatible with doppler solves.
 - Extended the automated test suite to check docstring examples.
-- Added an analytic 1D doppler-averaged steady-state solver `doppler_1d_exact`.
-  This solver is significantly faster for Doppler-averaged solves.
-  For now, this solver is considered experimental.
+- Added an analytic 1D doppler-averaged steady-state solver doppler_1d_exact. This solver is significantly faster for Doppler-averaged solves. For now, this solver is considered experimental.
```

Comment:
We can keep the code ticks here. Linking doesn't happen unless you use `:func:`.